src/current/cockroachcloud/create-a-serverless-cluster.md

```diff
@@ -62,45 +65,43 @@ All CockroachDB {{ site.data.products.cloud }} organizations get 50M RUs and 10
 
 <section class="filter-content" markdown="1" data-scope="free">
 
-1. Select the **Start for free** option.
+1. On the **Capacity** page, select the **Start for free** option.
 
     {{site.data.alerts.callout_info}}
     This will only be available if you haven't already created a free CockroachDB {{ site.data.products.serverless }} cluster or set up billing information.
     {{site.data.alerts.end}}
 
+1. Click **Next: Finalize**.
+
 </section>
 
 <section class="filter-content" markdown="1" data-scope="paid">
 
-1. If the option to **Start for free** is still available to you, select **Upgrade your resources** instead.
+1. On the **Capacity** page, if the option to **Start for free** is still available to you, select **Upgrade your resources** instead.
 
 1. Enter your **Resource limits**.
-    - If you select **Set a monthly limit**, you can set storage and RU limits individually, or enter a dollar amount that will be split automatically between both resources. You will only be charged for the resources you use.
     - If you select **Unlimited**, your cluster will scale to meet your application's needs. You will only be charged for the resources you use.
+    - If you select **Set a monthly limit**, you can set storage and RU limits individually, or enter a dollar amount that will be split automatically between both resources. You will only be charged for the resources you use.
 
-</section>
-
-## Step 5. Name the cluster
-
-The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes). A cluster's name cannot be edited after it is created.
-
-If you're creating a free cluster or you've already set up your billing information, click **Create cluster**. Your cluster will be created in a few seconds.
+1. Click **Next: Finalize**.
 
-If you still need to set up billing information, click **Next: Payment**.
+</section>
 
-## Step 6. Enter billing details
+## Step 5. Enter billing details
 
-1. On the **Summary** page, verify your cluster configuration and [resource limits](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#resource-limits).
+1. On the **Finalize** page, verify your cluster configuration and [capacity](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#resource-limits).
 
     {{site.data.alerts.callout_info}}
     The cost displayed does not include taxes.
```

Comment:
Consider: it might be worth reiterating here that this cost is a (maximum) estimate based on the capacity you've selected. But the actual cost would be based on the capacity you use. It's already mentioned throughout the doc, so not super important.